src/Common/src/Interop/Unix/System.Native/Interop.MountPoints.cs

```diff
@@ -0,0 +1,303 @@
+using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Sys
+    {
+        private const int MountPointFormatBufferSizeInBytes = 32;
+
+        /// <summary>
+        /// Internal FileSystem names and magic numbers taken from man(2) statfs
+        /// </summary>
+        /// <remarks>
+        /// These value names MUST be kept in sync with those in GetDriveType below
+        /// </remarks>
+        internal enum LinuxFileSystemTypes : long
```

Comment:
Is this still the right name for the type?  UnixFileSystemTypes, or just FileSystemTypes?